### PR TITLE
Takes away emag from medical borgs, and gives it to sabo borgs.

### PR DESCRIPTION
**Pull request by Red-byte3D**
## About The Pull Request
Takes away emg from medi borgs and gives it to sabo borgs
## Why It's Good For The Game
Doesnt necessarily make sense that the borg whos supposed to heal you has a emag, but the one whos supposed to go around sabotaging stuff does not.

also for some reason sabo borgs did not have an analyzer now they do
## Changelog
:cl:
fix: Syndicate Saboteur borgs now properly come with a analyzer installed
balance: Syndicate Medical borgs no longer have an emag
balance: Syndicate Saboteur borgs now come with an emag.
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
no

**Comment by InsaneRed** [comment]
> no

is your only argument no or are you gonna elaborate

**Comment by ATH1909** [comment]
> > no
> 
> is your only argument no or are you gonna elaborate

I'm about to, hold on

**Comment by ATH1909** [comment]
Suppose you're a nuke op team leader. You opt to not declare war, and instead opt to have your team buy exactly one item: a syndicate saboteur cyborg beacon. You summon the saboteur borg and move the ship onto the station's z-level.

The saboteur cyborg assumes an engiborg disguise and goes onto the station alone while the rest of the nuke op team has a soda chugging contest or something back on the ship. The saboteur cyborg finds the captain, stun arms them, move onto their tile, buckles the captain to themselves instantly, and beelines for the nearest airlock to space. The captain cries out for aid over comms, but nobody can help him- the syndicate saboteur cyborg does not suffer from damage slowdown, cannot be remotely blown or locked down by a fast-acting RD like an emagged engiborg can, and cannot be bolted in by the AI. Even if the HoS (with his unique gun set to ion mode) responds to the captain's call for help immediately and knows exactly where to go, he cannot outpace the saboteur cyborg without nuka cola or the like and reach it before it accomplishes its mission.

While it has been making its beeline to space, the syndicate saboteur cyborg has been weldering the captain to death, with some extra stun armings mixed in until he falls into crit, just in case. It uses its rods (or its ion thrusters if it gets those; I forget if it does) to speedbridge over to the nuke op shuttle, where the ops then strip the captain and arm the nuke with his nuke disk. The round is pretty much over at this point- an op (or even the aforementioned saboteur cyborg) drags the nuke over to the station, anchors it in maint or solars or whatever, ejects the disk, and returns to the nuke op ship with the disk in hand. The nuke ops leave the station's z-level on their ship, then spin and clap on the syndicate base while the station explodes.

**Comment by ATH1909** [comment]
tldr; The syndicate saboteur cyborg is not given a stun arm (or even an emag that it could emag a station-side cyborg with) for a _damn good reason_, and it's the same reason that nuke ops can't buy a (functional) chameleon kit or sleepy pens.

**Comment by InsaneRed** [comment]
> tldr; The syndicate saboteur cyborg is not given a stun arm (or even an emag that it could emag a station-side cyborg with) for a _damn good reason_, and it's the same reason that nuke ops can't buy a (functional) chameleon kit or sleepy pens.

i see your point about the arm even if its very hpyothetical and i heavily doubt something like this would happen and if it did that person would need to probably be really robust and fucking at that point just props to him so i suppose i can revert the arm change but im defending emag because you just said it yourself, if the ai doesnt have eyes on his borg and doesnt realize it is emagged then thats just ais fault and rds fault for not acting fast. Also emagging the experi sci machine and gibbing something is very funny

**Comment by ATH1909** [comment]
> > tldr; The syndicate saboteur cyborg is not given a stun arm (or even an emag that it could emag a station-side cyborg with) for a _damn good reason_, and it's the same reason that nuke ops can't buy a (functional) chameleon kit or sleepy pens.
> 
> i see your point about the arm even if its very hpyothetical and i heavily doubt something like this would happen and if it did that person would need to probably be really robust and fucking at that point just props to him so i suppose i can revert the arm change but im defending emag because you just said it yourself, if the ai doesnt have eyes on his borg and doesnt realize it is emagged then thats just ais fault and rds fault for not acting fast. Also emagging the experi sci machine and gibbing something is very funny

you don't need to be robust to do the arm plan. you literally just need to don an engiborg disguise, find the captain, ask him a question, and then click on him while he's typing, and you've basically already won.

**Comment by IndieanaJones** [comment]
Don't care about the arm, but the emag change should happen.

**Comment by InsaneRed** [comment]
> > > tldr; The syndicate saboteur cyborg is not given a stun arm (or even an emag that it could emag a station-side cyborg with) for a _damn good reason_, and it's the same reason that nuke ops can't buy a (functional) chameleon kit or sleepy pens.
> > 
> > 
> > i see your point about the arm even if its very hpyothetical and i heavily doubt something like this would happen and if it did that person would need to probably be really robust and fucking at that point just props to him so i suppose i can revert the arm change but im defending emag because you just said it yourself, if the ai doesnt have eyes on his borg and doesnt realize it is emagged then thats just ais fault and rds fault for not acting fast. Also emagging the experi sci machine and gibbing something is very funny
> 
> you don't need to be robust to do the arm plan. you literally just need to don an engiborg disguise, find the captain, ask him a question, and then click on him while he's typing, and you've basically already won.

but this is already assuming

Theres a captain,
the "captain" has secured the disk
and the entirety of station is unrobust enough to stop one borg with ranged ions

**Comment by InsaneRed** [comment]
> > > tldr; The syndicate saboteur cyborg is not given a stun arm (or even an emag that it could emag a station-side cyborg with) for a _damn good reason_, and it's the same reason that nuke ops can't buy a (functional) chameleon kit or sleepy pens.
> > 
> > 
> > i see your point about the arm even if its very hpyothetical and i heavily doubt something like this would happen and if it did that person would need to probably be really robust and fucking at that point just props to him so i suppose i can revert the arm change but im defending emag because you just said it yourself, if the ai doesnt have eyes on his borg and doesnt realize it is emagged then thats just ais fault and rds fault for not acting fast. Also emagging the experi sci machine and gibbing something is very funny
> 
> you don't need to be robust to do the arm plan. you literally just need to don an engiborg disguise, find the captain, ask him a question, and then click on him while he's typing. you've basically already won after that point.

and also you are claiming the ai is not fast enough to notice a unsycnhed borg

**Comment by ATH1909** [comment]
> but this is already assuming
> 
> Theres a captain,
> the "captain" has secured the disk
> and the entirety of station is unrobust enough to stop one borg with ranged ions

the syndicate ship has a camera console and the ops and the borg get syndicate encryption keys, y'know.

in order to use those ranged ions to stop the borg, they need to actually GET to them AND even HAVE those ranged ions 5 minutes into the round.

**Comment by InsaneRed** [comment]
> > but this is already assuming
> > Theres a captain,
> > the "captain" has secured the disk
> > and the entirety of station is unrobust enough to stop one borg with ranged ions
> 
> the syndicate ship has a camera console and the ops and the borg get syndicate encryption keys, y'know.
> 
> in order to use those ranged ions to stop the borg, they need to actually GET to them AND even HAVE those ranged ions 5 minutes into the round.

if the disk is not secured by the captain/acting captain then the station deserves to blow up
also you can unbuckle people from borgs now so assistant joe nuts can save the captain as well

**Comment by ATH1909** [comment]
> and also you are claiming the ai is not fast enough to notice a unsycnhed borg

yes, I am. if the AI does not notice the saboteur cyborg before it bops the captain, there is very little that it can do to stop it (effectively).

**Comment by InsaneRed** [comment]
> > and also you are claiming the ai is not fast enough to notice a unsycnhed borg
> 
> yes, I am. if the AI does not notice the saboteur cyborg before it bops the captain, there is very little that it can do to stop it (effectively).

you are really overestimating a borgs ability to open doors when the ai can spam at the same efficiency

**Comment by InsaneRed** [comment]
> > and also you are claiming the ai is not fast enough to notice a unsycnhed borg
> 
> yes, I am. if the AI does not notice the saboteur cyborg before it bops the captain, there is very little that it can do to stop it (effectively).

you have kind of convinced me on the arm thing i see your point i will wait for more feedback and then change it later on

**Comment by ATH1909** [comment]
> > > but this is already assuming
> > > Theres a captain,
> > > the "captain" has secured the disk
> > > and the entirety of station is unrobust enough to stop one borg with ranged ions
> > 
> > 
> > the syndicate ship has a camera console and the ops and the borg get syndicate encryption keys, y'know.
> > in order to use those ranged ions to stop the borg, they need to actually GET to them AND even HAVE those ranged ions 5 minutes into the round.
> 
> if the disk is not secured by the captain/acting captain then the station deserves to blow up
> also you can unbuckle people from borgs now so assistant joe nuts can save the captain as well

joe needs to actually be in melee range of the saboteur borg and not get gamer arm'd in order to unbuckle him.

the captain does not get unstunned when he gets unbuckled either, so even if he gets unbuckled from the saboteur cyborg taking damage or something, the sabby borg can just get onto his tile and rebuckle him to itself instantly.

**Comment by ATH1909** [comment]
> Don't care about the arm, but the emag change should happen.

the emag dissuades station-side cyborgs from feeling obligated by their laws to try to flash duel syndicate medical cyborgs (and syndicate assault cyborgs), as losing the duel will cause them to be turned against their comrades (a law 1 threat) instead of merely dying (a law 3 threat).

**Comment by InsaneRed** [comment]
> > > > but this is already assuming
> > > > Theres a captain,
> > > > the "captain" has secured the disk
> > > > and the entirety of station is unrobust enough to stop one borg with ranged ions
> > > 
> > > 
> > > the syndicate ship has a camera console and the ops and the borg get syndicate encryption keys, y'know.
> > > in order to use those ranged ions to stop the borg, they need to actually GET to them AND even HAVE those ranged ions 5 minutes into the round.
> > 
> > 
> > if the disk is not secured by the captain/acting captain then the station deserves to blow up
> > also you can unbuckle people from borgs now so assistant joe nuts can save the captain as well
> 
> joe needs to actually be in melee range of the saboteur borg and not get gamer arm'd in order to unbuckle him.
> 
> the captain does not get unstunned when he gets unbuckled either, so even if he gets unbuckled from the saboteur cyborg taking damage or something, the sabby borg can just get onto his tile and rebuckle him to itself instantly.

also if were going very hypothetical if you really wanted to do this all youd have to do is just buy an emag buy any borg, reset him with the tools you have make him go engi and emag him

**Comment by InsaneRed** [comment]
> > Don't care about the arm, but the emag change should happen.
> 
> the emag dissuades station-side cyborgs from feeling obligated by their laws to try to flash duel syndicate medical cyborgs (and syndicate assault cyborgs), as losing the duel will cause them to be turned against their comrades (a law 1 threat) instead of merely dying (a law 3 threat).
> 
> it also means that syndicate medical cyborgs don't have to nag an op to emag a cyborg that they have chainstunned.

mediborgs are not combat borgs and shouldnt leave ops side if they have a borg chainstunned they can just use their esword to kill it

**Comment by ATH1909** [comment]
> > > > > but this is already assuming
> > > > > Theres a captain,
> > > > > the "captain" has secured the disk
> > > > > and the entirety of station is unrobust enough to stop one borg with ranged ions
> > > > 
> > > > 
> > > > the syndicate ship has a camera console and the ops and the borg get syndicate encryption keys, y'know.
> > > > in order to use those ranged ions to stop the borg, they need to actually GET to them AND even HAVE those ranged ions 5 minutes into the round.
> > > 
> > > 
> > > if the disk is not secured by the captain/acting captain then the station deserves to blow up
> > > also you can unbuckle people from borgs now so assistant joe nuts can save the captain as well
> > 
> > 
> > joe needs to actually be in melee range of the saboteur borg and not get gamer arm'd in order to unbuckle him.
> > the captain does not get unstunned when he gets unbuckled either, so even if he gets unbuckled from the saboteur cyborg taking damage or something, the sabby borg can just get onto his tile and rebuckle him to itself instantly.
> 
> also if were going very hypothetical if you really wanted to do this all youd have to do is just buy an emag buy any borg, reset him with the tools you have make him go engi and emag him

```
/mob/living/silicon/robot/model/syndicate/ResetModel()
	return
```
you can't reset the model of a syndicate cyborg. in theory, you could deconstruct one and then rebuild it, but you'd have to replace its two flashes to do that. to do that without going to the station first, you'd need to spend 14 TC on hypnotic flashes, which (plus the 2 TC that you'd need to spend on the emag) is a considerablely larger investment of TC. also, this emagged engiborg can be remotely locked down and/or blown by anyone with robotics access and a robotics console who hears about this rogue cyborg.

**Comment by ATH1909** [comment]
> > > Don't care about the arm, but the emag change should happen.
> > 
> > 
> > the emag dissuades station-side cyborgs from feeling obligated by their laws to try to flash duel syndicate medical cyborgs (and syndicate assault cyborgs), as losing the duel will cause them to be turned against their comrades (a law 1 threat) instead of merely dying (a law 3 threat).
> > it also means that syndicate medical cyborgs don't have to nag an op to emag a cyborg that they have chainstunned.
> 
> mediborgs are not combat borgs and shouldnt leave ops side if they have a borg chainstunned they can just use their esword to kill it

*esaw.

they most certainly ARE combat cyborgs. their energy saw does esword damage, they get knockout chems, and they get a combat defib to knock down humans with.

**Comment by ATH1909** [comment]
and just because you're at an op's side doesn't mean that wrangling them to do basic tasks won't be absolute hell. we already struggle to get them to even weld us, please don't make us need to herd those cats any more than we already do.

**Comment by InsaneRed** [comment]
> and just because you're at an op's side doesn't mean that wrangling them to do basic tasks won't be absolute hell. we already struggle to get them to even weld us, please don't make us need to herd those cats any more than we already do.

if your team cant communicate and work together you deserve to lose some benefits

also they do not get knockout chems, and their defibs only knock you down so you can simply just put the flash in your bag and flash them afterwards

**Comment by ATH1909** [comment]
> > and just because you're at an op's side doesn't mean that wrangling them to do basic tasks won't be absolute hell. we already struggle to get them to even weld us, please don't make us need to herd those cats any more than we already do.
> 
> if your team cant communicate and work together you deserve to lose some benefits
> 
> also they do not get knockout chems, and their defibs only knock you down so you can simply just put the flash in your bag and flash them afterwards

morphine is a knockout chem, and the description of the hypospray they get explicitly states that the morphine is meant to be used offensively.

disarmfibs also deal 60 stamina damage, so you can back up, wait 5 seconds for your defib to recharge, and then disarmfib your victim again

**Comment by JoshAdamPowell** [comment]
Could the sabo borg get some kind of modified emag which doesnt work on borgs?

**Comment by ATH1909** [comment]
> Could the sabo borg get some kind of modified emag which doesnt work on borgs?

no matter how you sprite that, a saboteur is going to try to emag another borg with that item, think they succeeded, and then get ganked

**Comment by ATH1909** [comment]
I'm not, like, SUPER against saboteur cyborgs getting emags, but syndicate medical cyborgs shouldn't lose theirs

**Comment by Nexofas** [comment]
Stun arm aside, I support it. 
About the arm: instastuns on a borg type named "sabo" makes it more of a fighter rather than a saboteur considering the tools saboteur borgs have.

**Comment by Mothblocks** [comment]
> no matter how you sprite that, a saboteur is going to try to emag another borg with that item, think they succeeded, and then get ganked

the obvious answer is an audio cue to the user/borg for a successful emag

**Comment by Mothblocks** [comment]
anyway i didn't read anything else above this because it looked sort of boring but i don't like stun arms, your "fix" doesn't really explain why it's a fix, and i think things are fine as they are now

**Comment by Mothblocks** [comment]
They said they were gonna make changes.

**Comment by IndieanaJones** [comment]
OK this is a good PR now

**Comment by Mothblocks** [comment]
You need to update your changelog

**Comment by Mothblocks** [comment]
nvm
